specs/Terminate Spec.md

```diff
@@ -8,7 +8,7 @@
 
 **1.1. Elevator Pitch / Narrative** 
 
-Mike is debugging his code in Visual Studio and the program freezes with the "App not responding" text displaying on the title bar, informing him that he cannot continue his work without closing the program and restarting it. Mike can completely shutdown the app quicker using a Keyboard shortcut if he has this PowerToy installed on his computer.
+Mike is debugging his code in Visual Studio and the program freezes with the "App not responding" text displaying on the title bar, informing him that he cannot continue his work without closing the program and restarting it. Mike has tried to close the program using the close button and end task via the task manager, but none of these mechanisms work in helping him terminate the process so he can proceed with his work. With this PowerToy installed, Mike now has a visual and accessible last-resort method to help him kill the process.
```

Comment:
@miniksa do you think it's worded better to fit the description properly now?

Reply:
I love this description as it speaks to me and a problem I've actively had. Don't let me overshadow other customer feedback though.